Makefile

```diff
@@ -121,6 +121,15 @@ lint-go-code: $(GOLANGCI_LINT_BIN)
 $(GOLANGCI_LINT_BIN):
 	$(Q)curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ./out v1.17.1
 
+.PHONY: courier
+## Validate manifests using operator-courier
+courier:
+	$(Q)python3 -m venv ./out/venv3
+	$(Q)./out/venv3/bin/pip install --upgrade setuptools
+	$(Q)./out/venv3/bin/pip install --upgrade pip
+	$(Q)./out/venv3/bin/pip install operator-courier==2.0.2
```

Comment:
There is a new version of operator-courier: https://pypi.org/project/operator-courier/2.1.2/

Reply:
They are quite fast, on June 22 they released `2.1.1` and on June 24 `2.1.2`. Anyways I'll update it to latest one.

Reply:
I'll just put `pip install operator-courier` it should fetch latest one.